config/dispatcher/dispatcher.ex

```diff
@@ -580,6 +580,97 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://cache/distributions/"
   end
 
+  #################################################################
+  # Public Services - LPDC-IPDC-PoC
+  #################################################################
+  match "/conceptual-public-services/*path" do
+    Proxy.forward conn, path, "http://resource/conceptual-public-services/"
```

Comment:
I'm not 100% sure but I assume most of these can go through the cache service to improve performance?

Reply:
Indeed, using cache now